packages/document/docs/en/blog/release/release-note-0_4.mdx

```diff
@@ -0,0 +1,63 @@
+- **Easy to configure**: Enable by configuring the [mode.brief](/config/options/options#mode) option in the Rsdoctor plugin.
+- **Report output**: The report will be generated in the build artifacts directory, and the output directory and filename of the Brief report can be configured.
+
+This feature greatly enhances the convenience of using Rsdoctor analysis reports in CI processes. The report can be uploaded to a CDN in the pipeline to display historical reports. [For more details, please refer to](/guide/start/cicd)
+
+## Bundle Diff
+
+import { Badge } from '@theme';
+
+<Badge text="Version: 0.4.5" type="warning" />
+
+The **Bundle Diff** feature helps users compare and analyze changes between two build artifacts. Key capabilities include:
+
+- **Total size change**: Displays the total size change between two build artifacts.
+- **File type change**: Compares the size changes of different types of files (JS, CSS, images, fonts, etc.).
+- **First screen resource change**: Analyzes the size changes of resources loaded on the first screen.
+- **Duplicate package count change**: Counts and compares the number of duplicate packages between two builds.
+- **NPM Packages change**: Displays changes in the number of NPM packages, including added, removed, and changed packages.
+- **Detailed data query**: Provides a detailed list module for deeper data query and analysis.
+- **Module-level comparison**: View the name, size, and change rate of modules, and supports viewing the code changes of modules.
+
+<div style={{ display: 'flex', justifyContent: 'space-between' }}>
+  <img
+    src="https://raw.githubusercontent.com/rspack-contrib/rstack-design-resources/6c20c39a1dd576a513c2a7cb0fb0d2e162f63343/others/assets/rsdoctor/bundle-diff-cards.png"
+    style={{ width: '48%' }}
+  />
+  <img
+    src="https://raw.githubusercontent.com/rspack-contrib/rstack-design-resources/632e3087495b569de77a93235196fdacf09627d5/others/assets/rsdoctor/bundle-diff-module-list.png"
+    style={{ width: '48%' }}
+  />
+</div>
+
+This feature enhances the user's ability to visually analyze changes in build artifacts, helping users better understand and optimize build results. [For more details, please refer to](/guide/usage/bundle-diff)
+
+## Vue Loader Analysis
+
+<Badge text="Version: 0.4.5" type="warning" />
+
+Support for loader analysis for Vue projects has been added in Rsdoctor v0.4.
+
+<img src="https://raw.githubusercontent.com/rspack-contrib/rstack-design-resources/d8074588e99dbdcaa278cd1e82f1298663e425db/others/assets/rsdoctor/vue-loader.png" />
+
+## Other Features
+
+- **Rspack's builtin:lightningcss-loader analysis**: Added support for [builtin:lightningcss-loader](https://rspack.dev/guide/features/builtin-lightningcss-loader#builtin-lightningcss-loader) analysis.
+- **Performance optimization**:
+  - Reduced the size of the `@rsdoctor/client` package by 60%, improving page rendering speed.
```

Comment:
This can be highlighted and add a picture https://packagephobia.com/result?p=@rsdoctor/client@0.4.5

Reply:
DONE